dbs/mongo/4.0/Dockerfile

```diff
@@ -13,6 +13,8 @@ COPY data/load.sh /docker-entrypoint-initdb.d/
 # in the image so that they can be pulled from them for local testing.
 COPY certificates/metabase.* certificates/metamongo.* certificates/metaca.* /etc/mongo/
 
+COPY 4.0/mongo-with-ssl.sh /bin/
```

Comment:
Probably worth pointing out that we don't actually support MongoDB 4.0 any more it's EOL'ed upstream https://www.mongodb.com/support-policy/lifecycles

We should probably stop testing against it. 4.2 is the minimum version we support now